File: Python Code/Daily_Flash_Code_0/Week 3/31Jan/16-DailyFlash_Solutions/16-DailyFlash_Solutions/31_Jan_Solutions_Two/Python/progam3.py
try:
	n = int(input("Enter Size "))
except ValueError as e:
	exit(0)	
if(n <= 0):
	exit(0)

for i in range(1,n+1):
	for j in range(1,n-i+2):
		print(chr(64 + i), end = "\t")
	print()

File: Python Code/Daily_Flash_Code_0/Week 3/31Jan/16-DailyFlash_Solutions/16-DailyFlash_Solutions/31_Jan_Solutions_Two/C/progam3.c

#include <stdio.h>
#include <stdlib.h>

int main () {

	int n;
	printf("Enter Size : ");
	scanf("%d" , &n);

	if (n < 0)
		exit(0);

	for(int i = 1; i <= n; i++){
		for(int j = 1; j <= n-i+1; j++){
			printf("%c\t", 64 + i);
		}
		printf("\n");
	}
}
